spec/factories/conferences.rb

```diff
@@ -1,7 +1,9 @@
 FactoryGirl.define do
   factory :conference do
     name { [FFaker::CheesyLingo.title, 'Conf'].join ' ' }
-    tickets 2
+    starts_on { Time.utc(Date.today.year, 7, 7) }
```

Comment:
(Noticed the CheesyLingo lib in the FFaker gem ^ ? That was created by Team Cheesy during Soc15, and dedicated to SoC.) #thingsrailsgirlsdo